src/extension.ts

```diff
@@ -49,7 +49,7 @@ function activateMdExt(context: ExtensionContext) {
     // Allow `*` in word pattern for quick styling (toggle bold/italic without selection)
     // original https://github.com/microsoft/vscode/blob/3e5c7e2c570a729e664253baceaf443b69e82da6/extensions/markdown-basics/language-configuration.json#L55
     languages.setLanguageConfiguration('markdown', {
-        wordPattern: /([*_]{1,2})?[\p{Alphabetic}\p{Number}\p{Nonspacing_Mark}]+(_[\p{Alphabetic}\p{Number}\p{Nonspacing_Mark}]+)*\1/gu
+        wordPattern: /([*_]{1,2}|~~|`+)?[\p{Alphabetic}\p{Number}\p{Nonspacing_Mark}]+(_|[\p{Alphabetic}\p{Number}\p{Nonspacing_Mark}]+)*\1/gu
```

Comment:
Can you check whether

```diff
- /([*_]{1,2}|~~|`+)?[\p{Alphabetic}\p{Number}\p{Nonspacing_Mark}]+(_|[\p{Alphabetic}\p{Number}\p{Nonspacing_Mark}]+)*\1/gu
+ /([*_]{1,2}|~~|`+)?[\p{Alphabetic}\p{Number}\p{Nonspacing_Mark}]+(_+[\p{Alphabetic}\p{Number}\p{Nonspacing_Mark}]+)*\1/gu
```

works for you, otherwise it doesn't seem right.

Reply:
I have checked using *word pattern* presented and it works fine.

Reply:
Then let's go for the later one 👍

Reply:
It is believed that the work indicated has been completed. Please confirm these. :eyes: